wireguard-go-rust-wrapper/wireguard-go/bind_windows.go

```diff
@@ -82,11 +82,11 @@ type WinRingBind struct {
 	isOpen uint32
 }
 
-func NewDefaultBind() conn.Bind { return NewWinRingBind() }
+func NewDefaultBind() FullBind { return NewWinRingBind() }
 
-func NewWinRingBind() conn.Bind {
+func NewWinRingBind() FullBind {
 	if !winrio.Initialize() {
-		return conn.NewStdNetBind()
+		return NewStdNetBind()
```

Comment:
:thinking: could we log somehow which bind is being used? WinRingBind or StdNetBind? 

I believe this may have performance implications, and may cause headaches trying to understand them. If we would have them logged -> this would help immensely :). AFAIK, go code logging is configured so :crossed_fingers: should not be difficult to log it from here.